File: file_read/file_read.py
#!/usr/bin/python3
# -*- coding: utf-8 -*-
FILE = './eng-uk_web_2002_1M-words.txt'

#------------------------------------------------------------------
def read_file(name):
    with open(name, 'r', encoding = 'utf-8') as f:
        x = f.readlines()
    print('Lines: ', len(x))
#------------------------------------------------------------------
def read_file2():
    with open(FILE, 'r', encoding = 'utf-8') as f:
        x = f.readlines()
#------------------------------------------------------------------
def main():
    print(f'Python file reading "{FILE}"...')
    read_file(FILE)
    print('Python exit')
# -----------------------------------------------------------------
if __name__ == '__main__':
    main()

File: file_read/file_read.R
#!/usr/bin/Rscript --vanilla
# a program in R language
# R version 3.4.4 (2018-03-15)

FILENAME <-  './eng-uk_web_2002_1M-words.txt'

# --------------------------------------------------------------------------
read_file <- function(name) {
   cat('Reading txt file ', name, '... ')
   f <- file(name, 'r', encoding = 'UTF-8')
   lines <- readLines(f, encoding = 'UTF-8')
   close(f)
   cat('done!\n')
   cat('Lines:', length(lines), '\n')
}
# ------------------------- "main" (global) ---------------------------------
cat('R script started.\n')
read_file(FILENAME)
cat('R exit.\n')
